app/components/ui/molecules/FileUpload.js

```diff
@@ -84,18 +88,16 @@ const DropzoneContent = ({ conversion, formError, dropzoneOpen }) => {
     )
   }
   if (conversion.error) {
-    const errorMessage = get(
-      conversion,
-      'error.message',
-      'Error Uploading File',
-    )
     return (
       <React.Fragment>
         <StyledUploadFailureIcon />
-        <ErrorText data-test-id="dropzoneMessage">
-          {errorMessage}. Try to <Action onClick={dropzoneOpen}>upload</Action>{' '}
-          your manuscript again.
-        </ErrorText>
+        <Paragraph data-test-id="dropzoneMessage">
+          <DropzoneErrorText>Oops!</DropzoneErrorText> Unable to upload
+          manuscript. Please <Action onClick={dropzoneOpen}>try again.</Action>
+          <StyledParagraph>
+            Files should be in .docx or .pdf format and no larger than 100MB.
+          </StyledParagraph>
+        </Paragraph>
```

Comment:
Sorry if this is a WIP because you don't want comments yet? But just noticed that this `<Paragraph>` tag is sitting around the `StyledParagraph`, so you've got a `p` within a `p`

Reply:
Hm, I guess we should change to a div then? This was fine when it was a div and not a paragraph and I do want to group both messages together so it's easier to search for the the entire message (e.g. in tests) and wrapping everything in another div is too much nesting.